packages/core/strapi/lib/middlewares/compression.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const compress = require('koa-compress');
+
+/**
+ * @type {import('./').MiddlewareFactory}
+ */
+module.exports = (options = {}) => compress(options);
```

Comment:
Why do we need to encapsulate the call to compress here? (since it already handles nil options)

Reply:
We can but then the types signature would be different to the MiddlewareFactory one as we also pass and object with strapi even if it's not used. I'm happy to just return compress too tbh